account_invoice_bank_brand/tests/test_account_move.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountInvoiceBankBrand(TransactionCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
It's better to use `setUpClass` when possible.

Reply:
Done